bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -62,12 +62,12 @@ spec:
                                   description: 'sourcePath is the subpath within contextDir
                                     where kustomize builds start. Examples include
                                     any sub-folder or path: `base`, `overlays/dev`,
-                                    `default`, `odh` etc'
+                                    `default`, `odh` etc.'
```

Comment:
I've seen this in couple of PRs now... but #845 is merged

Reply:
think need rebase the code to get it